README.md

```diff
@@ -81,25 +88,38 @@ public readonly partial struct Hp : IEquatable<Hp> , IComparable<Hp>
     private class HpTypeConverter : System.ComponentModel.TypeConverter { /* snip... */ }
 
     // UnitGenerateOptions.ArithmeticOperator
-    public static Hp operator +(in Hp x, in Hp y) => new Hp(checked((int)(x.value + y.value)));
-    public static Hp operator -(in Hp x, in Hp y) => new Hp(checked((int)(x.value - y.value)));
-    public static Hp operator *(in Hp x, in Hp y) => new Hp(checked((int)(x.value * y.value)));
-    public static Hp operator /(in Hp x, in Hp y) => new Hp(checked((int)(x.value / y.value)));
+    public static Hp operator +(Hp x, Hp y) => new Hp(checked((int)(x.value + y.value)));
```

Comment:
The `in`  modifiers was removed because it could not comply with interfaces such as INumber<>.